pkg/controller/import-controller.go

```diff
@@ -489,6 +490,11 @@ func (r *ImportReconciler) createImportEnvVar(pvc *corev1.PersistentVolumeClaim)
 		podEnvVar.previousCheckpoint = getValueFromAnnotation(pvc, AnnPreviousCheckpoint)
 		podEnvVar.currentCheckpoint = getValueFromAnnotation(pvc, AnnCurrentCheckpoint)
 		podEnvVar.finalCheckpoint = getValueFromAnnotation(pvc, AnnFinalCheckpoint)
+
+		dv := &cdiv1.DataVolume{}
```

Comment:
So all communication between the data volume and a PVC goes through annotations. The established pattern for something like this is to write a 'preallocation' annotation on the PVC by the data volume controller when it creates the PVC. Then in the import/upload/clone controller we read the annotation. This way none of those controller need to know that data volumes even exist.